.travis.yml

```diff
@@ -48,4 +48,5 @@ script:
   - cd $TRAVIS_BUILD_DIR/backend/src
   - gimme -f 1.11.4
   - source ~/.gimme/envs/go1.11.4.env
+  - go vet -shadow ./...
```

Comment:
I think this means only run the shadow checks?
You might need to vet shadows separately on a second invokation, since the flag appears to be still experimental.

Reply:
Thanks, good catch! I verified that `go vet -all -shadow` runs all checks including shadow.